sandbox/apparmor/notify/permission.go

```diff
@@ -32,9 +32,9 @@ const (
 	AA_MAY_OPEN
 	// AA_MAY_RENAME implies that a process may rename a file.
 	AA_MAY_RENAME
-	// AA_MAY_SETATTR is not checked by the kernel.
+	// AA_MAY_SETATTR is not checked by the kernel but may appear in messages.
 	AA_MAY_SETATTR
-	// AA_MAY_GETATTR is not checked by the kernel.
+	// AA_MAY_GETATTR is not checked by the kernel but may appear in messages.
```

Comment:
same not as with setattr but it is the read permission being mapped.

Reply:
How about `AA_MAY_GETCRED` and `AA_MAY_SETCRED`? Should these be exposed to the user as `"read"` and `"write"` permissions for the `"home"` interface, or should we deny these if they come through for home?

Reply:
No. do not map "getcred" and "setcred"